src/main/resources/static/css/layout.css

```diff
@@ -0,0 +1,58 @@
+.entryBody{
```

Comment:
going with css naming conventions, this should be something like `entry-body` (hyphenated instead of camelCase or PascalCase. This same rule applies to #id's usually)